net/node/transactionPool.go

```diff
@@ -5,98 +5,310 @@ import (
 	"DNA/common/log"
 	"DNA/core/ledger"
 	"DNA/core/transaction"
-	"DNA/errors"
-	msg "DNA/net/message"
-	. "DNA/net/protocol"
+	"DNA/core/transaction/payload"
+	va "DNA/core/validation"
 	"fmt"
 	"sync"
 )
 
 type TXNPool struct {
 	sync.RWMutex
-	txnCnt uint64
-	list   map[common.Uint256]*transaction.Transaction
+	txnCnt        uint64                                      // count
+	txList        map[common.Uint256]*transaction.Transaction // transaction which have been verifyed will put into this map
```

Comment:
here all the transaction abbreviation is txn instead of tx to avoid mess up with transmit abbreviation tx.